apm-commons/apm-util/src/main/java/org/apache/skywalking/apm/util/MethodUtil.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.util;
+
+import java.lang.reflect.Method;
+
+/**
+ * According to the input parameter,
+ * return the OperationName for the span record,
+ * It can determine the unique method
+ *
+ * @author zhaoyuguang
+ */
+
+public class MethodUtil {
```

Comment:
This is not SkyWalking util. It is an agent util. Please move it into your plugin

Reply:
Then, let us move it into agent-core/util